README.md

```diff
@@ -1,2 +1,37 @@
-# manage_crm_objects
-Sample application in PHP, Python, Ruby and JavaScript demonstrating HubSpot API to manage CRM Objects
+# HubSpot CRM Objects sample app
+
+This is a sample app for the HubSpot [client libraries](https://developers.hubspot.com/docs/api/overview). This sample app demonstrates how to authenticate your HubSpot app using OAuth 2.0, and make an API call to different CRM Objects API.
```

Comment:
Can we remove unrelated to this app description?

Reply:
Why it's unrelated?

Reply:
"This sample app demonstrates how to authenticate your HubSpot app using OAuth 2.0" - it doesn't, coz uses hapikey instead of oauth

Reply:
oh, you mean phrase about oauth, got it